1-mandatory/2-people-i-know.js

```diff
@@ -393,7 +395,15 @@ You can see who people's colleagues are by seeing the "colleagues" array in each
 This time, I only want the full names ("<firstname> <lastname>") of my friends who are colleagues of hers.
 */
 
-let friendsWhoAreColleaguesOfStacie = [];
+let friendsWhoAreColleaguesOfStacie = friends
+  .filter(
```

Comment:
The logic seems good but look for a way to optimize the code. Mainly look at other loops implementation that can help you in this case.